snackbars/README.md

```diff
@@ -0,0 +1,66 @@
+# TextInput components
+
+## TextInput design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:snackbars:<versions>")
+```
+
+### Primary
+
+```kotlin
+object VitaminSnackbars {
+    @Composable
+    fun Primary(
+        text: String,
+        modifier: Modifier = Modifier,
+        actionOnNewLine: Boolean = false,
+        colors: SnackbarColors = VitaminSnackbarColors.primary(),
+        action: @Composable (VitaminSnackbarsActions.() -> Unit)? = null
+    ) = Snackbar(
```

Comment:
You shouldn't show the implementation (like other components), just the contract.